mypy/plugin.py

```diff
@@ -384,24 +384,48 @@ class DefaultPlugin(Plugin):
 
     def get_function_hook(self, fullname: str
                           ) -> Optional[Callable[[FunctionContext], Type]]:
+        from mypy.plugins import ctypes
+
```

Comment:
> Another idea is to move the `ctypes` plugin to `plugin.py` since all other stdlib plugins live there

That would be possible, but I'm not sure if it's worth it here. The circular import issues aren't too big (the only affected parts are the argument type annotations of the hooks). And since the `ctypes` plugin is a bit larger than the other hooks in `plugin.py`, I think it makes more sense to have it in a separate module. (It's not the only stdlib plugin to have its own module - `dataclasses` is separated as well, and it's roughly the same size as the `ctypes` plugin.)